src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1630,14 +1630,42 @@ TEST_P(activation_random_test, random) {
     execute_compare(param, true);
 }
 
-INSTANTIATE_TEST_SUITE_P(activation_blocked_tests,
-                         activation_random_test,
-                         testing::ValuesIn(
-                            std::vector<activation_random_test_params>{
-                                { data_types::i8,  format::b_fs_yx_fsv32,        { 1, 32, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::i8,  format::bs_fs_yx_bsv32_fsv32, {32, 32, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::f16, format::bs_fs_yx_bsv32_fsv16, {32, 32, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::i8,  format::bs_fs_yx_bsv32_fsv32, {16, 16, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::f16, format::bs_fs_yx_bsv32_fsv16, {16, 16, 5, 5}, activation_func::relu, {}, {}},
-                            }
-                        ));
+INSTANTIATE_TEST_SUITE_P(
+    activation_blocked_tests,
+    activation_random_test,
+    testing::ValuesIn(std::vector<activation_random_test_params>{
+        {data_types::i8, format::b_fs_yx_fsv32, {1, 32, 5, 5}, activation_func::relu, {}, {}},
+        {data_types::i8, format::bs_fs_yx_bsv32_fsv32, {32, 32, 5, 5}, activation_func::relu, {}, {}},
+        {data_types::f16, format::bs_fs_yx_bsv32_fsv16, {32, 32, 5, 5}, activation_func::relu, {}, {}},
+        {data_types::i8, format::bs_fs_yx_bsv32_fsv32, {16, 16, 5, 5}, activation_func::relu, {}, {}},
+        {data_types::f16, format::bs_fs_yx_bsv32_fsv16, {16, 16, 5, 5}, activation_func::relu, {}, {}},
+        // gelu-7
+        {data_types::f16, format::bfyx, {16, 16, 5, 5}, activation_func::gelu, {}, {}},
+        {data_types::f16, format::bfyx, {16, 16, 5, 5}, activation_func::gelu_tanh, {}, {}},
+        {data_types::f32, format::bfyx, {16, 16, 5, 5}, activation_func::gelu, {}, {}},
+        {data_types::f32, format::bfyx, {16, 16, 5, 5}, activation_func::gelu_tanh, {}, {}},
```

Comment:
can't you use combine inside? 
Like in example [here](https://github.com/openvinotoolkit/openvino/blob/a544fc70d189fa4b1fed85ebb08bf584ae5649de/src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/non_max_suppression.cpp#L30)?

Reply:
Done